pages/login.tsx

```diff
@@ -115,6 +120,21 @@ export default function Login() {
       }
     }
   }, [$email, $setLoaded, $queryEmail, $queryAutoLogin, submit])
+  const updatedContext =
+    $toast === 'You have been logged out.'
+      ? {
+          checkLoggedIn: () => false,
+          checkLoading: () => false,
+          checkFailed: () => false,
+          status: STATUS.LOGGED_OUT,
+          setStatus: () => null,
+          error: null,
+          magicMetadata: null,
+          metadata: null,
+        }
+      : loginContext
```

Comment:
This is a hack -- we should update the source of the data to a logged-out state when the logout happens rather than covering up the data and switching on the contents of the toast.

Reply:
I ended up adding these two state resets: https://github.com/iron-fish/website-testnet/pull/126/files#diff-ccc3bc2a49ec13caa0caab60e351ddd83198bd11352f1d051b7c70876e284950R22-R23 to replace this change.